### PR TITLE
Changed bollinger styling to new color scheme

### LINKED ISSUE #184: Bollinger Band Styling
Red for Upper
Green for Lower
Grey for Fill

See google doc.

### PR DESCRIPTION
**Pull request by djmiley**
Resolves #184

### CODE REVIEW

File: src/assets/styles/bollinger.less [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.plot-area {
```
Comment (jleft) [inline src/assets/styles/bollinger.less]: I think these would be fine in `plotArea.less` - we can always add comments, so we know which rules relate to which series/indicator.

File: src/assets/styles/bollinger.less [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  .average > path.line{
```
Comment (jleft) [inline src/assets/styles/bollinger.less]: Space between `line` and `{` - same for below too.

File: src/assets/styles/variables.less [hunk 3]
```diff
@@ -9,6 +9,11 @@
 @candlestick-down-fill: #c60;
 @candlestick-up-fill: #6c0;
 
+// Bollinger
```
Comment (jleft) [inline src/assets/styles/variables.less]: Again, I'd probably group under the plot area heading. Or split up the plot area variables into annotation, moving average and candlestick.

File: src/assets/styles/variables.less [hunk 4]
```diff
@@ -9,6 +9,11 @@
...
+@bollinger-upper-colour: #0f0;
```
Comment (jleft) [inline src/assets/styles/variables.less]: Would it make sense to use the `#c60` and `#6c0` (from candlestick) as the upper and lower colours - then we'd have consistent shades of red/green? Might look rubbish though - I'm not sure!

**Comment by djmiley** [comment]
Yes, happy to change the PR for those suggestions

File: src/assets/styles/variables.less [hunk 5]
```diff
@@ -2,15 +2,18 @@
...
 
 // Plot Area
-@annotation-text-colour: #fff;
-@moving-average-line-colour: #f0f;
 @candlestick-down-stroke: #000;
 @candlestick-up-stroke: #000;
 @candlestick-down-fill: #c60;
 @candlestick-up-fill: #6c0;
+@moving-average-line-colour: #f0f;
+@bollinger-upper-colour: #6c0;
```
Comment (jleft) [inline src/assets/styles/variables.less]: Maybe pull `#6c0` as another variable - `@primary-green`? And `#c60` - `@primary red`?

Should the colours be the other way around? #184:

> Red for Upper
Green for Lower

Comment (djmiley) [inline src/assets/styles/variables.less]: Doh.

**Comment by jleft** [comment]
:+1:

**Pull request merged**